nle/scripts/plot.py

```diff
@@ -103,27 +118,36 @@ def str_to_float_pair(s):
 )
 
 
-def plot_single_ascii(target, width, height, window=-1, xrange=None, yrange=None):
+def plot_single_ascii(
+    target,
+    width,
+    height,
+    xkey="# Step",
```

Comment:
No need for the default values to show up here again. Let's remove these and take the default from the flags.

Reply:
This was in case the function was imported outside of a parser call. I would have thought everything but target could be defaulted for an import

Happy to change this - but some arguments have defaults, some dont

Reply:
True enough. Let's keep it like this for now.